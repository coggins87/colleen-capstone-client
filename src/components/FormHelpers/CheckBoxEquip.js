import React from 'react'

export const CheckBoxEquip = props => {
    return (
      <li>
       <input key={props.id} onClick={props.handleCheck} type="checkbox" checked={props.isChecked} value={props.value} /> {props.value}
      </li>
    )
}

export default CheckBoxEquip