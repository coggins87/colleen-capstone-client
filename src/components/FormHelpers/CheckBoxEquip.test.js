import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import CheckBoxEquip from './CheckBoxEquip'

describe('CheckBoxEquip component', () => {
  const props = {
    className: 'test-class-name',
    testCheck: ()=>{},
    'data-other': 'test-other-prop'
  }

  it('renders a CheckBoxEquip by default', () => {
    const wrapper = shallow( <CheckBoxEquip />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })

  it('renders the CheckBoxEquip given props', () => {
    const wrapper = shallow( <CheckBoxEquip {...props} />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})