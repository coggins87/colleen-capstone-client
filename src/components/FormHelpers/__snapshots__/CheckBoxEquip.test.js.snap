// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckBoxEquip component renders a CheckBoxEquip by default 1`] = `
<Fragment>
  <input
    type="checkbox"
  />
   
</Fragment>
`;

exports[`CheckBoxEquip component renders the CheckBoxEquip given props 1`] = `
<Fragment>
  <input
    type="checkbox"
  />
   
</Fragment>
`;
